src/components/page-header/page-header.md

```diff
@@ -0,0 +1,12 @@
+```
```

Comment:
Can we have some direction here about how to use the component as part of a larger page layout? Maybe specify that only one should be used per page?

Reply:
On it. I'll try to add some accessibility stuff too.